ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.12-nightly-x86.yaml

```diff
@@ -22,7 +22,8 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: control-plane-etcd-fips-24nodes
+- always_run: false
+  as: control-plane-etcd-fips-24nodes
   cron: 0 8 23 * *
```

Comment:
I am expecting validation errors here to have both always_run: false and a cron schedule, as I thought the two were mutually exclusive.

Reply:
Done.